src/components/orders/OrderFormContainer.js

```diff
@@ -1,18 +1,25 @@
-import React, { useState } from 'react';
-import { Form, Input, Button, Tooltip } from 'antd';
-
-import { QuestionCircleOutlined } from '@ant-design/icons';
+import React, { useState, useEffect } from 'react';
+import { Form, Input, Button, InputNumber } from 'antd';
 
 import 'antd/dist/antd.css';
 
 const OrderFormContainer = () => {
   const [form] = Form.useForm();
   const [formLayout, setFormLayout] = useState('horizontal');
+  const [, forceUpdate] = useState();
 
   const onFormLayoutChange = ({ layout }) => {
     setFormLayout(layout);
   };
 
+  useEffect(() => {
+    forceUpdate({});
+  }, []);
+
+  const onFinish = values => {
+    console.log('Finish:', values);
```

Comment:
Just a console.log

Reply:
Noted. Will replace with axios call after this pull request has been merged. Want to keep the buyer action and reducers consistent with the state management already built.